hepcrawl/testlib/tasks.py

```diff
@@ -19,11 +19,8 @@
 
 
 class Config(object):
-    CELERY_RESULT_BACKEND = "amqp://guest:guest@rabbitmq:5672//"
-    BROKER_URL = "amqp://guest:guest@rabbitmq:5672//"
-    CELERY_ALWAYS_EAGER = True
```

Comment:
I'm surprised the tests don't require some flavor of this, but they seem to pass, so it must be OK.

Reply:
This happens to work because of the Celery Monitor, which essentially attaches receivers that listen to signals like "a task completed successfully": https://github.com/inspirehep/hepcrawl/blob/b753c2250f27b649ad5c903ff1d3c0de9229c5ac/hepcrawl/testlib/celery_monitor.py#L76-L82.

BTW, as remarked in https://github.com/inspirehep/hepcrawl/pull/241, `hepcrawl` has been tested for a long time against Celery 4, and the name of this parameter changed to `task_always_eager`, so this configuration option has had no effect for who knows how much time.